src/couch_index_updater.erl

```diff
@@ -202,15 +202,16 @@ purge_index(Db, Mod, IdxState) ->
             {ok, IdxState};
         true ->
             FoldFun = fun({PurgeSeq, _UUId, Id, Revs}, Acc) ->
-                Mod:purge(Db, PurgeSeq, [{Id, Revs}], Acc)
+                {ok, StateAcc} = Mod:purge(Db, PurgeSeq, [{Id, Revs}], Acc),
+                StateAcc
             end,
             {ok, NewStateAcc} = couch_db:fold_purged_docs(
-                    Db,
```

Comment:
Thanks, Bob. The format issue was fixed here intentionally to use same indent mode.